src/Languages/Twig/Injections/TwigPropertyInjection.php

```diff
@@ -0,0 +1,30 @@
+<?php
+
+declare(strict_types=1);
+/**
+ * Created by PhpStorm.
+ * User: Jozef Môstka
+ * Date: 24. 3. 2024
+ * Time: 6:56
+ */
+
+namespace Tempest\Highlight\Languages\Twig\Injections;
+
+use Tempest\Highlight\Highlighter;
+use Tempest\Highlight\Injection;
+use Tempest\Highlight\IsInjection;
+
+class TwigPropertyInjection implements Injection
+{
+    use IsInjection;
+
+    public function getPattern(): string
+    {
+        return '(?<match>({{(.|\n)*?}}))';
+    }
+
+    public function parseContent(string $content, Highlighter $highlighter): string
+    {
+        return $highlighter->parse($content, 'twig');
```

Comment:
One idea I was playing around with, is to have "sub languages", so that you can have a more specific pattern set for specific parts of your language. For example, the property injection could be parsed like so:

```php
return $highlighter->parse($content, new TwigPropertyLanguage());
```

I'm not sure it'd be beneficial enough. And you definitely don't need to change it within this PR. I just wanted to mention it and see what your thoughts were on it.

Reply:
Yes, this will be better. because in TwigCommentInjection is chid html language, and in that language is twig. but now I cand do that. twig language is not highlighted inside twig comment. but this can be also fixed if twig language will be extended HTML language.
See TwigLanguageTest line 31. there is not higlighted twig language inside comment

Reply:
Ok thanks for that feedback! I'm going to merge this PR as is, and then improve on it some more on main :)